apps/dashboard/templates/meinberlin_dashboard/base_user.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Dashboard" %}{% endblock %}
+
+{% block content %}
+
+<div class="l-wrapper">
+    <div class="menu-layout">
+
+        <div class="menu-layout__menu">
+            <div class="dropdown">
```

Comment:
No. It's done analogue to https://github.com/liqd/a4-meinberlin/pull/149/files#diff-0f97d36510376a473f11473ce48afa0cR38 In case there is no dropdown there'd still need to be a title of some sorts.